### ISSUE TITLE
[BUG} Tree printing produces disconnected lines

### ISSUE THREAD

**Issue by auronen** [clementwanjau/tree-ds#14]
**Describe the bug**
The printing seems to have a bug with node level - producing visualisation with cosmetic bugs.

**To Reproduce**
Seems to be a problem when the first level after root node has more than one child with its own children.

**Expected behavior**
 (tree snippet)
```
Union.vdf
├─ _WORK
│  └─ DATA
│     ├─ ANIMS
│     │  └─ _COMPILED
│     │     ├─ HUMANS_SKELETON-T_SPAWN.MAN
│     │     ├─ INVALID_SOURCE_FILE.MDL
│     │     ├─ INVALID_SOURCE_FILE.MDM
│     │     └─ INVALID_SOURCE_FILE.MMB
│     ├─ MESHES
│     │  └─ _COMPILED
│     │     └─ INVALID_SOURCE_FILE.MRM
│     ├─ SOUND
│     │  ├─ INVALID_SOURCE_FILE.OGG
│     │  └─ INVALID_SOURCE_FILE.WAV
│     └─ TEXTURES
│        └─ _COMPILED
│           └─ INVALID_SOURCE_FILE-C.TEX
└─ SYSTEM
   ├─ ASMCL.DLL
```
current version produces:

```
Union.vdf
├── _WORK
│   └── DATA
│       ├── ANIMS
    │       └── _COMPILED
    │           ├── HUMANS_SKELETON-T_SPAWN.MAN
    │           ├── INVALID_SOURCE_FILE.MDL
    │           ├── INVALID_SOURCE_FILE.MDM
    │           └── INVALID_SOURCE_FILE.MMB
│       ├── MESHES
    │       └── _COMPILED
    │           └── INVALID_SOURCE_FILE.MRM
│       ├── SOUND
    │       ├── INVALID_SOURCE_FILE.OGG
    │       └── INVALID_SOURCE_FILE.WAV
│       └── TEXTURES
    │       └── _COMPILED
    │           └── INVALID_SOURCE_FILE-C.TEX
└── SYSTEM
    ├── ASMCL.DLL
```

**Comment by clementwanjau** [comment]
Hello @auronen .

Thank you for bringing this to my attention. I'll give it a look ASAP.

Cheers

**Comment by clementwanjau** [comment]
This has been fixed in v0.1.6.
